### ISSUE TITLE
Cropped image is stretched to the full canvas size

### ISSUE THREAD

**Issue by jakeorr** [MattMcFarland/react-darkroom#12]
It should maintain it's aspect ratio with white borders the same way it does when you load an image.

**Comment by MattMcFarland** [comment]
Yes I believe I glossed past this issue a while back because I was not able to to successfully achieve this.  This was quite a while ago, so maybe I could figure it out now since I have leveled up since I originally wrote this library.  

The issue I ran into I believe (going from memory) was the ratio calculations.  I seemed to have got stuck there, I dont know the details.  I'll take a look and at least if I cant solve the issue, perhaps I'll raise it on stack overflow or ask for help once I can at least take a crack at it again and try to figure out what is wrong.
